tools/claim_output_base.sh

```diff
@@ -20,27 +20,30 @@ if [ ! -z "${BAZEL_OUTPUT_BASE-}" ]; then
   return
 fi
 
+if [ -d "${HOME}/.cache/bazel/_bazel_${USER}" ]; then
+  SUFFIX="$(pwd | md5sum | head -c6)"
+  OUTPUT_ROOT="${HOME}/.cache/bazel/_bazel_${USER}"
+else
+  SUFFIX="$(pwd | md5 | head -c6)"
+  OUTPUT_ROOT="/private/var/tmp/_bazel_${USER}"
```

Comment:
why not just `/var/tmp/...` in this case?

Reply:
`/private/var/tmp` is what is in the bazel docs and what `bazel info output_root` returns. I don't know how they come up with it (the code is `/var` but presumably they readlink or something somewhere) but I also don't really want to know -- I just want to take what they _are_ using and append something.

Reply:
does that work for you?

Reply:
i had no objection the first time around. i have already approved.